eclair-core/src/test/scala/fr/acinq/eclair/testutils/MySlf4jLogger.scala

```diff
@@ -0,0 +1,142 @@
+package fr.acinq.eclair.testutils
+
+import akka.actor.{Actor, ExtendedActorSystem}
+import akka.dispatch.RequiresMessageQueue
+import akka.event.Logging.{Debug, Error, Info, InitializeLogger, LogEvent, LogEventWithCause, LogEventWithMarker, LoggerInitialized, Warning}
+import akka.event.slf4j.{SLF4JLogging, Slf4jLogMarker}
+import akka.event.{DummyClassForStringSources, LoggerMessageQueueSemantics}
+import akka.util.Helpers
+import ch.qos.logback.classic.LoggerContext
+import org.slf4j.{ILoggerFactory, MDC, Marker, MarkerFactory}
+
+/**
+ * Copy of [[akka.event.slf4j.Slf4jLogger]], with a customizable [[ILoggerFactory]] so we can separate logs in tests.
```

Comment:
Unfortunately Akka's [`Slf4jLogger`](https://github.com/akka/akka/blob/v2.6.18/akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala) doesn't allow us to select a particular `LoggingContext`, hence I had to slightly tweak it.